code/modules/uplink/uplink_items/uplink_bundles.dm

```diff
@@ -193,6 +193,8 @@
 			var/datum/uplink_item/I = uplink_items[category][item]
 			if(src == I || !I.item)
 				continue
+			if(istype(I, /datum/uplink_item/bundles_TC/reroll)) //oops!
+				continue
```

Comment:
Actually, just remove this entry from the possible_items list after the for loop is done, more efficient until a new variable to blacklist items from the pick random choice is made.

Reply:
Hm, unsure on if that'd actually be more efficient than just moving this check a few lines down right before the adding it to the list, as checking every item that's going to be added to the list would have a worst-case of n, whereas having to go through the entire list afterwards (because it's saved as items, not typepaths, so can't just use the -= operator directly), then removing the specifc item would have a worstcase of n^2, n being the amount of suitable items.

Reply:
It seems I didn't make myself understood. What I meant to suggest was actually removing the reroll item after the whole `for` loop is done, but apparently I didn't notice the `possible_items` keys are instances and not typepaths or text strings.

So TL;DR this is fine as is, I was distracted.